Common/DTO/RestRequests/EditUserRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace DTO.RestRequests
+{
+    public class EditUserRequest
```

Comment:
I might have missed something, but should not we have id somewhere in request to search user which is being changed? Or it's some other model?

Reply:
Yeah, I think you are right

Reply:
fixed